### ISSUE TITLE
Register JSON Schemas with schemastore.org

### ISSUE THREAD

**Issue by aaronsteers** [dbt-labs/dbt-jsonschema#8]
Thanks so much for creating these! Have you considered registering these with https://schemastore.org?

At [Meltano](meltano.com), we registered our JSON schema in schemastore.org - which is then automatically applied in VS Code and other IDEs that subscribe to the schemastore.org manifests (listed at https://www.schemastore.org/json/#editors).

Our PR is here as an example: https://github.com/SchemaStore/schemastore/pull/2020

Within a few hours of the PR being merged, VS Code users will start getting auto-complete and underline-linting automatically with no local config required.  🚀  (The only requirement, I think, is that they have the Yaml extension installed.)

**Comment by joellabes** [comment]
It's definitely in the cards! See #3 - because we have multiple schema files, to make them play nicely together they'd have to be uploaded into the schemastore repo instead of just referencing the GitHub hotlink, but that has negative tradeoffs too. 

(Of course without #4  the "playing nice together" is purely hypothetical)

**Comment by gwenwindflower** [comment]
i want to bump this issue now that we have the `latest` schema definition set up, we should be able to tackle this right?

feels like we should get it on the roadmap.

**Comment by joellabes** [comment]
Agreed! The one thing that has been holding me back is that there is nothing about a dbt project's `whatever.yml` files that identifies them as dbt files, which means we would need to lay claim to the entire `**.yml` glob and I don't think the schemastore folks would like that. 

I guess we could assume that everyone's directories follow the defaults and claim [`**/models/**.yml`, `**/seeds/**.yml`, `**/snapshots/**.yml`] and so on as a solid guess.

**Comment by gwenwindflower** [comment]
i like it, i think that's solid, while i think people definitely get creative with how they organize yaml files within those directories, i doubt too many people deviate from that.

**Comment by will-sargent-dbtlabs** [comment]
Hi yall, jumping on the thread here... If we assume the defaults (in favor of), is it possible for folks on the VSC side to manually add additional directories to assume the files are ours?  If so, then it feels like covering the 80% or better is a low-friction choice, that still leaves the options open for those that have non-standard top-level folders..

**Comment by joellabes** [comment]
Yes that’s correct! They can either continue to configure manually in `.vscode/settings.json` or associate it through the VS Code UI

**Comment by yu-iskw** [comment]
@joellabes I don't completely understand there are any blockers to register the schemas to schemastore.org. But, I am tring to add them. If there is anything you are woried about, please let me know.

https://github.com/SchemaStore/schemastore/pull/3948

**Comment by aaronsteers** [comment]
I think this is shipped! 

- https://github.com/SchemaStore/schemastore/pull/3977

Closing - Thanks, @yu-iskw and @joellabes!
